packages/web-components/examples/all-charts.html

```diff
@@ -240,6 +240,17 @@ <h2>Map</h2>
       childPlaceType="State"
       variable="Count_Person"
     ></datacommons-map>
+    <datacommons-map
+      parentPlace="Earth"
+      childPlaceType="Country"
+      variable="Count_Person"
+    ></datacommons-map>
+    <datacommons-map
+      parentPlace="Earth"
+      childPlaceType="Country"
+      variable="Count_Person"
+      cacheGeoJsonProp="unGeoCoordinate"
```

Comment:
nit: can we call it unGeoJsonCoordinates?

In line with https://datacommons.org/browser/geoJsonCoordinates